Editor/AvatarRigCreator.cs

```diff
@@ -0,0 +1,148 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Animations.Rigging;
+
+namespace ReadyPlayerMe.Core.Editor
+{
+    public abstract class AvatarRigCreator
+    {
+        private const string TAG = nameof(AvatarRigCreator);
+
+        private const string ANIMATION_RIGGING_PACKAGE_NAME = "com.unity.animation.rigging";
+        private const string DIALOG_TITLE = "Read Player Me";
+        private const string DIALOG_MESSAGE = "This will require installing the Animation Rigging package. Do you want to continue?";
+        private const string DIALOG_OK = "Ok";
+        private const string DIALOG_CANCEL = "Cancel";
+
+        private const float HINT_OFFSET_Z = 0.08f;
+
+        [MenuItem("Ready Player Me/Add Avatar Rig")]
+        public static void AddAvatarRig()
+        {
+            var selectedObjects = Selection.gameObjects;
+            var avatars = selectedObjects.Where(x => x.GetComponent<AvatarData>() != null).ToList();
+
+            if (avatars.Any())
+            {
+                GetAnimationRiggingPackage();
+            }
+
+            foreach (var avatar in avatars)
+            {
+                SetupRig(avatar);
+            }
+        }
+
+        private static void SetupRig(GameObject avatar)
+        {
+            AddBoneRenderer(avatar);
+
+            var rig = AddRigBuilder(avatar);
+
+            SetupRightHand(avatar.transform, rig.transform);
+            SetupLeftHand(avatar.transform, rig.transform);
+            SetupHead(avatar.transform, rig.transform);
+        }
+
+        private static void AddBoneRenderer(GameObject avatar)
+        {
+            var boneRenderer = avatar.AddComponent<BoneRenderer>();
+            var hips = FindTransform(avatar.transform, "Hips");
+            var children = GetAllChildren(hips);
+            children.Add(hips);
+            boneRenderer.transforms = children.ToArray();
+        }
+
+        private static GameObject AddRigBuilder(GameObject avatar)
+        {
+            var rigBuilder = avatar.AddComponent<RigBuilder>();
+            var rigPrefab = AssetDatabase.LoadAssetAtPath<GameObject>("Assets/Ready Player Me/Core/Editor/Prefabs/Rig.prefab");
```

Comment:
I think this path won't work when a package is installed into Packages folder (outside of Unity-SDK project)? 
Probably need to handle that.